apps/website-new/docs/en/practice/monorepos/index.mdx

```diff
@@ -0,0 +1,99 @@
+# Benefits of Module Federation in a Monorepo
+
+Module Federation is a powerful tool that allows you to share code between applications. It enables you to build applications that are more modular, maintainable, and scalable. In this guide, we will explore the benefits of using Module Federation in a monorepo.
+
+Polyrepos (multiple repositories) have long since been the de facto choice for developing micro-frontends, to combat organizational issues, however, they introduce more organizational challenges around code sharing, dependency management, versioning and team autonomy.
+
+Enabling team autonomy via Micro Frontends inadvertently pushed teams towards a polyrepo structure. Each team would have their own repository and would be responsible for maintaining their own codebase. This approach led to a lack of collaboration and a lack of transparency among other challenges.
+
+You can get all the benefits of Module Federation without the hassle of managing multiple repositories when you choose a monorepo, including some additional benefits around code sharing and dependency management.
+This guide will help you understand the benefits of using Module Federation in a monorepo and how they address organizational challenges.
+
+## Flaws with the Polyrepo Approach
```

Comment:
Maybe instead of flaws we call this "Potential drawbacks of a polyrepo approach"